guacamole/src/main/frontend/src/translations/en.json

```diff
@@ -183,6 +183,79 @@
 
     },
 
+    "IMPORT": {
+        
+        "ACTION_ACKNOWLEDGE"          : "@:APP.ACTION_ACKNOWLEDGE",
+
+        "BUTTON_CANCEL": "Cancel",
+        "BUTTON_CLEAR": "Clear",
+        "BUTTON_IMPORT": "Import Connections",
```

Comment:
The prefix for actions that may be taken, button or not, is `ACTION_`. Here, for generic actions like "Cancel" and "Clear", they should be defined at the `APP` level and then referenced.